pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -1126,7 +1107,7 @@ func registerClusterToCluster(r registry.Registry) {
 			timeout:            3 * time.Hour,
 			additionalDuration: 60 * time.Minute,
 			cutover:            30 * time.Minute,
-			clouds:             registry.AllExceptAWS,
```

Comment:
If you end up having to push a fix for the lint failure, I'll note that the TODO here seems out of date.